src/tsickle.ts

```diff
@@ -217,18 +222,64 @@ function addClutzAliases(
 
   // .d.ts files can be transformed, too, so we need to compare the original node below.
   const origSourceFile = ts.getOriginalNode(sourceFile);
-  // The module exports might be re-exports, and in the case of "export *" might not even be
-  // available in the module scope, which makes them difficult to export. Avoid the problem by
-  // filtering out symbols who do not have a declaration in the local module.
+  // In order to write aliases, the exported symbols need to be available in the
+  // the module scope. That is not always the case:
+  //
+  // 1) export const X;           // works
```

Comment:
nit: (1) (2) (3) below is a bit hard to understand, maybe name these "export", "reexport" and "imported reexport"?

Reply:
My hunch was that the confusion comes from us being sloppy with the words 'export' and 'reexport'. Does 'export' include a 'reexport', etc? Anyways, gave it a show, using only words, not sure if it is clearer. Time will tell.